### PR TITLE
Implement jump for IWMO and IWSV

### PR DESCRIPTION
**Pull request by gregorwolf**

### CODE REVIEW

File: src/objects/zcl_abapgit_object_iwmo.clas.abap [hunk 1]
```diff
@@ -90,7 +90,37 @@ CLASS ZCL_ABAPGIT_OBJECT_IWMO IMPLEMENTATION.
 
   METHOD zif_abapgit_object~jump.
 
-    zcx_abapgit_exception=>raise( |TODO: Jump| ).
+    DATA: lv_mdl_technical_name TYPE /iwbep/med_mdl_technical_name,
+          lv_version            TYPE /iwbep/med_mdl_version,
```
Comment (christianguenter2) [inline src/objects/zcl_abapgit_object_iwmo.clas.abap]: Do these structures in systems lower 740 without embedded gateway exist?

File: src/objects/zcl_abapgit_object_iwmo.clas.abap [hunk 2]
```diff
@@ -90,7 +90,37 @@ CLASS ZCL_ABAPGIT_OBJECT_IWMO IMPLEMENTATION.
...
+    APPEND INITIAL LINE TO lt_bdcdata ASSIGNING <ls_bdcdata>.
+    <ls_bdcdata>-fnam = 'GS_MODEL_SCREEN_100-VERSION'.
+    <ls_bdcdata>-fval = lv_version.
+
+    ls_opt-dismode = 'E'.
+    ls_opt-defsize = 'X'.
+
+    TRY.
+        CALL TRANSACTION '/IWBEP/REG_MODEL' WITH AUTHORITY-CHECK
+                                USING lt_bdcdata OPTIONS FROM ls_opt.
```
Comment (christianguenter2) [inline src/objects/zcl_abapgit_object_iwmo.clas.abap]: Is `WITH AUTHORITY-CHECK` 702 compliant?

File: src/objects/zcl_abapgit_object_iwmo.clas.abap (hunk 1 above)
Comment (gregorwolf) [reply src/objects/zcl_abapgit_object_iwmo.clas.abap]: That might be a problem yes.

File: src/objects/zcl_abapgit_object_iwmo.clas.abap (hunk 2 above)
Comment (gregorwolf) [reply src/objects/zcl_abapgit_object_iwmo.clas.abap]: Has someone a system to check it?

Comment (larshp) [reply src/objects/zcl_abapgit_object_iwmo.clas.abap]: I think its okay, https://syntax.abaplint.org/?filter=trans#/statement/CallTransaction

File: src/objects/zcl_abapgit_object_iwmo.clas.abap (hunk 1 above)
Comment (larshp) [reply src/objects/zcl_abapgit_object_iwmo.clas.abap]: strictly they are not needed? just move the value directly into `fval`? with offset and length fixes

**Comment by larshp** [comment]
will close for now, no progress, moving to issue https://github.com/larshp/abapGit/issues/3116

**Pull request closed without merging**